package.json

```diff
@@ -80,5 +80,8 @@
   "engines": {
     "node": ">=18.x",
     "npm": ">=9.x"
+  },
+  "dependencies": {
+    "ethereum-cryptography": "^2.1.3"
```

Comment:
The root `package.json` shouldn't have dependencies. These should be place in the corresponding package's `package.json`.